app/views/shared/organisations/_grant_conditions.html.erb

```diff
@@ -0,0 +1,15 @@
+<% if organisation.claims_grant_conditions_accepted_at %>
+  <%= govuk_summary_list(html_attributes: { id: "grant-conditions" }) do |summary_list| %>
+    <% summary_list.with_row do |row| %>
+      <% row.with_key(text: t(".agreed_by")) %>
+      <% row.with_value(**details_field_args(organisation.claims_grant_conditions_accepted_by&.full_name)) %>
+    <% end %>
+    <% summary_list.with_row do |row| %>
+      <% row.with_key(text: t(".agreed_on")) %>
+      <% row.with_value(**details_field_args(l(organisation.claims_grant_conditions_accepted_at, format: :long_no_time))) %>
```

Comment:
```suggestion
      <% row.with_value(**details_field_args(l(organisation.claims_grant_conditions_accepted_at.to_date, format: :long))) %>
```

Or, my personal preference:

`_at` being a time.
`_on` being a date, or the date-casted value of a time.

```ruby
class Claims::School < School
  def grant_conditions_accepted_on
    claims_grant_conditions_accepted_at&.to_date
  end
end
```